pkg/moov/wallet_test.go

```diff
@@ -30,6 +31,48 @@ func TestWalletMarshal(t *testing.T) {
 	require.Equal(t, "ec7e1848-dc80-4ab0-8827-dd7fc0737b43", wallet.WalletID)
 }
 
+func TestListWallet(t *testing.T) {
+	mc := NewTestClient(t)
+
+	wallets, err := mc.ListWallets(context.Background(), "ebbf46c6-122a-4367-bc45-7dd555e1d3b9")
+	NoResponseError(t, err)
+	require.NotNil(t, wallets)
+
+	for i := range wallets {
+		t.Logf("wallet[%d]: %#v", i, wallets[i])
+	}
+}
+
+func TestGetWallet(t *testing.T) {
```

Comment:
Locally I am seeing the List wallets call pass and give me a wallet (which I used in the Get Wallet test). 

The Get Wallet and List Transactions calls are failing. The API docs do not mention any additional auth required over the List Wallets call. 

- https://docs.moov.io/api/sources/wallets/list/
- https://docs.moov.io/api/sources/wallets/get/
- https://docs.moov.io/api/sources/wallets/list-transactions/